kube_scheduler/metadata.csv

```diff
@@ -0,0 +1,38 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+kube_scheduler.apiserver.audit.events,count,,,,"Number of audit events generated and sent to the audit backend",0,kube_controller_manager,apiserver.audit.events
```

Comment:
The integration name needs to be updated on this line, and a few others

Reply:
yes